### DISCUSSION TITLE
Pico-W-Go broken?

### DISCUSSION THREAD

**Discussion by FlydGitHub** [micropython/micropython#11334] [RP2040 / Pico]
Is Pico-W-Go broken?

Current Version of Vscode - as of Tus April 25 2023

**Comment by scruss** [comment]
best to ask the developer on their own site: [paulober/Pico-W-Go: Pico-W-Go is a Visual Studio code extension designed to simplify and speed up the development of MicroPython projects for the Raspberry Pi Pico and Pico W boards.](https://github.com/paulober/Pico-W-Go)

**Comment by FlydGitHub** [comment]
Hi Thanks Stuart,

 I can see how to contact the Developers.

On person said using an older veserion worked for him: That worked for me (now using v3.0.0) - thanks *so* much - no more Thonny, yay!
